SignalServiceKit/src/Util/OWSFileSystem.m

```diff
@@ -32,6 +58,33 @@ + (BOOL)protectFileOrFolderAtPath:(NSString *)path
     return YES;
 }
 
++ (void)logAttributesOfItemAtPath:(NSString *)path
+{
+    NSDictionary<NSFileAttributeKey, id> *_Nullable attributes = [self attributesOfItemAtPath:path];
+    DDLogDebug(@"%@ path: %@ has attributes: %@", self.logTag, path, attributes);
+}
+
++ (nullable NSDictionary<NSFileAttributeKey, id> *)attributesOfItemAtPath:(NSString *)path
+{
+    return [[NSFileManager defaultManager] attributesOfItemAtPath:path error:nil];
```

Comment:
Should we use an `NSError` and verify it?

Reply:
Done!